data/sles4sap/qe_sap_deployment/qesap_azure_peering.yaml

```diff
@@ -0,0 +1,54 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Maintainer: QE-SAP <qe-sap@suse.de>
+# Summary: Generic yaml template for use with qe-sap-deployment project: https://github.com/SUSE/qe-sap-deployment
+provider: 'azure'
+apiver: 3
+terraform:
+  variables:
+    az_region: '%REGION%'
+    deployment_name: '%DEPLOYMENTNAME%'
+    os_image: '%OS_VER%'
+    public_key: '%SLES4SAP_PUBSSHKEY%'
+    hana_remote_python: '%ANSIBLE_REMOTE_PYTHON%'
+    iscsi_remote_python: '%ANSIBLE_REMOTE_PYTHON%'
+    vnet_address_range: '%MAIN_ADDRESS_RANGE%'
+    subnet_address_range: '%SUBNET_ADDRESS_RANGE%'
+    ibsm_rg: '%IBSM_RG%'
```

Comment:
Two extra variables here:
```
ibsm_rg: '%IBSM_RG%'
    ibsm_vnet_name: '%IBSM_VNET%'
```

We can potentially add these two in all the qesap regression conf.yaml as, not providing the openQA variables,  they get empty value taht is fine for terraform code